### PR TITLE
Change rules snippet, add delwarn and !ban save option

### PR DESCRIPTION
**Pull request by TheTimeWalker**

### CODE REVIEW

File: mod/docs/index.md [hunk 1]
```diff
@@ -32,13 +32,14 @@
 | Windows Insiders (Dev)          | See the Commands section                                              | Receive notifications                                           |
 | 💻 Patch Tuesday                        | See the Commands section                                              | Receive notifications                                           |
 ## Commands
-Moderation commands are available through the MEE6 bot. Some commands are restricted to permanent moderators.
+Moderation commands are available through MicrosoftBot and Dyno. Some commands are restricted to permanent moderators.
```
Comment (Jasius) [inline mod/docs/index.md:35]: `MicrosoftBot` should be changed to `Cliptok`

File: member/index.md [hunk 2]
```diff
@@ -94,7 +94,8 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
 ### Misc Commands
 | Command                 | Function                                                 |
 | ----------------------- | -------------------------------------------------------- |
-| !rules                  | Posts a link to the server rules                         |
+| !rules                  | Posts a snippet of all rules                             |
```
Comment (Jasius) [inline member/index.md:97]: This actually wouldn't work rigth now, it requires `all` argument, but I am going to change it soon™, so that it works.

**Review by Jasius — CHANGES_REQUESTED** [review]

File: member/index.md (hunk 2 above)
Comment (Jasius) [reply member/index.md:97]: You can keep this as is.

Comment (TheTimeWalker) [reply member/index.md:97]: Oh! I haven't noticed that this changed but it made sense as it was a very big snippet. I've kept this for now though

**Review by Jasius — APPROVED** [review]

**Pull request merged**